client/src/client.ts

```diff
@@ -2300,9 +2301,15 @@ export abstract class BaseLanguageClient {
 		this._telemetryEmitter = new Emitter<any>();
 		this._stateChangeEmitter = new Emitter<StateChangeEvent>();
 		this._tracer = {
-			log: (message: string, data?: string) => {
-				this.logTrace(message, data);
-			}
+			log: (...params: any[]) => {
```

Comment:
Can this not still be message & data and message can be string | any. ...params makes it harder to understand what data can actually come.